### DISCUSSION TITLE
You're not allowed to use this command!

### DISCUSSION THREAD

**Discussion by Ch0p** [danthe1st/Mee6-bypasser#47] [Support] [answered]
When I want to use a command I get "You're not allowed to use this command!" 
What can I do?

**Comment by noobidev** [comment, marked as answer]
Make sure that your roles allow you to manage roles (`MANAGE_ROLES` permission) explicitly, administrator is not sufficient due to the implementation of our command framework.

**Reply by Ch0p** [reply]
The bot has the permission, but it still doesn't work

**Reply by noobidev** [reply]
You and the bot need `MANAGE_ROLES` in order for commands to work.
